src/chapters/spring/exercises.rst

```diff
@@ -0,0 +1,38 @@
+Exercises: Bonjour Spring!
```

Comment:
the exercises should just have the same name as the section title they are in since we've followed that convention in other areas

Reply:
Done! Thanks!